src/library/Text/styled.js

```diff
@@ -121,7 +128,16 @@ export const TextRoot = styled('p', {
       ...getCommonStyles(as, theme, truncate),
       // 1 - Not normalized because we actually want `##em` as applied value
       // 2 - Must come after commonStyles
-      marginBottom: noMargins ? 0 : theme.Text_marginBottom
```

Comment:
1. I wouldn't remove this line. Just set noMargins = true by default.
2. As a fallback, we can also set the Text_marginBottom on the theme to 0.